pycqed/measurement/waveform_control/circuit_builder.py

```diff
@@ -335,4 +335,19 @@ def seq_from_ops(self, operations, fill_values=None,  pulse_modifs=None,
                                       pulse_modifs=pulse_modifs).build()
         pulses += self.mux_readout(**ro_kwargs).build()
         seq.add(Segment('Segment1', pulses))
-        return seq
\ No newline at end of file
+        return seq
+
+    def simultaneous_blocks(self, block_name, blocks):
+        simultaneous = Block(block_name, [])
```

Comment:
minimal doc string would be nice for new features :). 
Here I think it would be particularly useful because the behavior is not so straightforward.
I would suggest somthing like:
```
"""
Creates a block with name :block_name: that consists of the parallel execution of the given :blocks:. Ensures that any pulse or block following the created block will occur after the longest given block. 

Note that within each of the given block, it is assumed that the pulse positioned last in the block is the one that occurs last. TODO: We might want to relax this assumption in a future version!
Args:
    block_name (string): name of the block that is created
    blocks (iterable): iterable where each element is a block that has to be executed in parallel to the others.
"""
```

Reply:
Thank you for the docstring! I will of course include it.